griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
```

Comment:
Should probably not have defaults.

For other cases when we do want the default to be `None`, let's always wrap the type into `Optional`. In this particular case it would be `Optional[str]`.

Reply:
Makes sense! I'll update the params to use Optional, and create a validator to check that one of `cluster_identifier` or `workgroup_name` are supplied